src/mscorlib/shared/Interop/Windows/Kernel32/Interop.SetLastError.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        [DllImport(Libraries.Kernel32, SetLastError = true)]
+        internal static extern void SetLastError(int lastError);
```

Comment:
Interesting I checked with @luqunl yesterday and said the marshaling code not setting last error to 0. I'll remove SetLastError(0) and put some comment on the code too. 

CC @yizhang82